From akpm@linux-foundation.org Tue Mar  6 02:48:14 2007
From: Simon Arlott <simon@arlott.org>
Date: Tue, 06 Mar 2007 02:47:46 -0800
Subject: USB: MAINTAINERS: cxacru
To: greg@kroah.com
Cc: linux-usb-devel@lists.sourceforge.net, akpm@linux-foundation.org, simon@arlott.org, baldrick@free.fr, simon@fire.lp0.eu
Message-ID: <200703061047.l26Alk3O020189@shell0.pdx.osdl.net>


From: Simon Arlott <simon@arlott.org>

I've acquired a second device for testing and plan to make some changes in
the near future to export all the device stats to sysfs (based on my
proposed patch to add them to the proc file ~2007-01-30).

Signed-off-by: Simon Arlott <simon@fire.lp0.eu>
Acked-by: Duncan Sands <baldrick@free.fr>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    5 +++++
 1 file changed, 5 insertions(+)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -952,6 +952,11 @@ M:	mhw@wittsend.com
 W:	http://www.wittsend.com/computone.html
 S:	Maintained
 
+CONEXANT ACCESSRUNNER USB DRIVER
+P:	Simon Arlott
+M:	cxacru@fire.lp0.eu
+S:	Maintained
+
 COSA/SRP SYNC SERIAL DRIVER
 P:	Jan "Yenya" Kasprzak
 M:	kas@fi.muni.cz
